src/pages/wishlist-page/wishlist-page.css

```diff
@@ -0,0 +1,27 @@
+.wishlist-card{
+  border:1px dotted gray;
+  height:350px;
+  width:250px;
+
+}
+.wishlist-container{
+  padding: 10px 10px;
+  width:250px;
+  display:flex;
```

Comment:
Suggestion: if values are less than 10 px use px only , other than that you can  rems